modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/portlet/RolesAdminPortlet.java

```diff
@@ -822,7 +857,12 @@ private boolean _isDepotGroup(long companyId, String groupKey) {
 	private ItemSelector _itemSelector;
 
 	private PanelAppRegistry _panelAppRegistry;
+	private ServiceTrackerList
+		<PanelCategoryRoleTypeMapper, PanelCategoryRoleTypeMapper>
+			_panelCategoryPermissionMapperServiceTrackerList;
```

Comment:
I think this can be renamed to `_ panelCategoryRoleTypeMapperServiceTrackerList` to avoid confusion. When I first see this variable, I thought it was tracking a different service.

Reply:
Good catch, that was the original class name 👍